### PR TITLE
Add acknowledgment of software GMT relies on

### PR DESCRIPTION
**Pull request by PaulWessel**
The usual list is now added to the end of the README in a new  acknowledgment section.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,3 +88,21 @@ Copyright (c) 1991-2020 by [the GMT Team](AUTHORS.md).
...
+GMT may be linked with these libraries (some are optional):
+[Geospatial Data Abstraction Library (GDAL)](https://gdal.org),
+[Network Common Data Form (netCDF)](https://www.unidata.ucar.edu/software/netcdf/),
+[Perl Compatible Regular Expressions (PCRE)](https://www.pcre.org), 
+[Fastest Fourier Transform in the West (FFTW)](http://www.fftw.org),
+[Linear Algebra Package (LAPACK)](http://www.netlib.org/lapack/),
+[Basic Linear Algebra Subprograms (BLAS)](http://www.netlib.org/blas/), and
+[ZLIB](https://www.zlib.net). GMT may call these executables:
+GDAL (ogr2ogr, gdal_info), [Ghostscript](https://www.ghostscript.com),
+[FFmpeg(https://www.ffmpeg.org)],
```
Comment (seisman) [inline README.md]:
```suggestion
[FFmpeg](https://www.ffmpeg.org),
```

File: README.md [hunk 2]
```diff
@@ -88,3 +88,21 @@ Copyright (c) 1991-2020 by [the GMT Team](AUTHORS.md).
...
+[dxg-open](https://www.freedesktop.org/wiki/Software/xdg-utils/), and
```
Comment (seisman) [inline README.md]:
```suggestion
[xdg-open](https://www.freedesktop.org/wiki/Software/xdg-utils/), and
```

File: README.md [hunk 3]
```diff
@@ -88,3 +88,21 @@ Copyright (c) 1991-2020 by [the GMT Team](AUTHORS.md).
...
+
+## Acknowledgment
+
+GMT relies on several other Open Source software libraries and programs for its
+operation.  We gratefully acknowledge the importance to GMT of these products.
+GMT may be linked with these libraries (some are optional):
+[Geospatial Data Abstraction Library (GDAL)](https://gdal.org),
+[Network Common Data Form (netCDF)](https://www.unidata.ucar.edu/software/netcdf/),
+[Perl Compatible Regular Expressions (PCRE)](https://www.pcre.org), 
+[Fastest Fourier Transform in the West (FFTW)](http://www.fftw.org),
```
Comment (joa-quim) [inline README.md]: We cannot say that we rely on FFTW (we don't). FFTW is GPL, we don't rely on GPL libs.

Comment (PaulWessel) [reply README.md]: Agree that we dont rely on it, we may use it.  I dont understand the point about GPL and that we dont rely on GPL libs.  What do you mean?

Comment (joa-quim) [reply README.md]: If we say we rely on a GPL lib, GMT has to be GPL.

Comment (PaulWessel) [reply README.md]: what is the approved verb then? use?

Comment (PaulWessel) [reply README.md]: Please help phrase this so we can get it done.

Comment (joa-quim) [reply README.md]: I think it will do if we add an [optional] at the end of all of them but netCDF.

Comment (PaulWessel) [reply README.md]: OK, I marked most as optional.  OK with you?

**Comment by PaulWessel** [comment]
See if you have comments on this, @joa-quim and @seisman.  The GMT splash screen now says something like this (my build):

```
	GMT - The Generic Mapping Tools, Version 6.1.0_3630ae8-dirty_2020.02.09 [64-bit] [8 cores]
	[Linked with netCDF, GDAL, PCRE, FFTW, LAPACK, ZLIB]
	(c) 1991-2020 The GMT Team (https://www.generic-mapping-tools.org/team.html).
```

File: README.md (hunk 3 above)
Comment (joa-quim) [reply README.md]: > OK, I marked most as optional. OK with you?

It's not me. I don't give a shit on it. It's the GNU zealots that I'm concerned with.

Comment (PaulWessel) [reply README.md]: It seems the standard practice is to ignore this stuff until a zealots starts attacking.  Then one can add both GPL and LPGL licenses to GMT and users can select which one, apparently.
Seems OK to you @seisman?

Comment (seisman) [reply README.md]: OK to me.

**Pull request merged**